modules/appeals_api/config/schemas/v2/10182_with_shared_refs.json

```diff
@@ -1,122 +1,108 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "description": "JSON Schema for VA Form 10182",
-  "$ref": "#/definitions/nodCreate",
-  "definitions": {
-    "nodCreate": {
+  "properties": {
+    "data": {
       "type": "object",
       "properties": {
-        "data": {
+        "type": { "type": "string", "enum": ["noticeOfDisagreement"] },
+        "attributes": {
           "type": "object",
           "properties": {
-            "type": { "type": "string", "enum": ["noticeOfDisagreement"] },
-            "attributes": {
+            "veteran": {
               "type": "object",
               "properties": {
-                "veteran": {
-                  "type": "object",
-                  "properties": {
-                    "homeless": { "type": "boolean" },
-                    "address": { "$ref": "address.json" },
-                    "phone": {
-                      "allOf": [
```

Comment:
While modifying these schema_with_shared_ref files, I also removed these extraneous `allOf` items and replaced them with simpler objects with the same behavior.